test/numa/run.sh

```diff
@@ -0,0 +1,479 @@
+#!/bin/bash
+
+DEMO_TITLE="CRI Resource Manager: Numa test"
+
+PV='pv -qL'
+
+binsrc=${binsrc-local}
+
+SCRIPT_DIR="$(dirname "${BASH_SOURCE[0]}")"
+DEMO_LIB_DIR=$(realpath "$SCRIPT_DIR/../../demo/lib")
+BIN_DIR=${bindir-$(realpath "$SCRIPT_DIR/../../bin")}
+OUTPUT_DIR=${outdir-$SCRIPT_DIR/output}
+COMMAND_OUTPUT_DIR=$OUTPUT_DIR/commands
+
+source $DEMO_LIB_DIR/command.bash
+source $DEMO_LIB_DIR/host.bash
+source $DEMO_LIB_DIR/vm.bash
+
+script_source="$(< "$0") $(< "$DEMO_LIB_DIR/vm.bash") $(< "$DEMO_LIB_DIR/command.bash")"
```

Comment:
nitpick: although host.bash does not seem to have functions with help ATM, wouldn't including it here anyway be safer in case we ever add such functions there ?

Reply:
Good one, fixed!